<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgBoxDlg</class>
 <widget class="QDialog" name="MsgBoxDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Pic">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Pic</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_info">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Infomation</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Arial&quot;;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Arial&quot;;
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 0);
font: 20pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>pushButton_OK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
